### DISCUSSION TITLE
Identify boards to be ported to Zephyr

### DISCUSSION THREAD

**Discussion by balajikulkarni** [zephyrproject-rtos/zephyr#50435] [Q&A] [answered]
Hi,

Is it safe to assume that the boards not included in https://docs.zephyrproject.org/3.0.0/boards/index.html#boards are not yet ported and can be taken up for porting to Zephyr? 

If not, how can I identify the boards yet to be ported to Zephyr not to overlap with developers working on porting the same board?

Thanks!

**Comment by DerekSnell** [comment, marked as answer]
Hi @balajikulkarni ,

The doc link you shared is for Zephyr release v3.0.0.  You should use the latest doc page at https://docs.zephyrproject.org/latest/boards/index.html#boards, which will list all the boards in the latest main branch.

But that will only list the boards that have been merged into the main branch.  If you are concerned about others also porting the same board, you could monitor the open Pull Requests to watch if others are working to add a new board.

Hope that helps.

**Reply by balajikulkarni** [reply]
Thanks @DerekSnell !
